app/src/main/java/es/wolfi/utils/GeneralUtils.java

```diff
@@ -0,0 +1,77 @@
+package es.wolfi.utils;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.design.widget.Snackbar;
+import android.util.Log;
+import android.view.View;
+import android.widget.Toast;
+
+public class GeneralUtils {
+
+    private static Toast lastToast;
+
+    private static void showtoast(Context c, @NonNull CharSequence message)
+    {
+        Log.d("GeneralUtils", message.toString());
+        if (lastToast != null)
+            lastToast.setText(message);
+        else {
+            lastToast = Toast.makeText(c, message, Toast.LENGTH_SHORT);
+        }
+        lastToast.show();
+    }
+
+    public static void debug(@NonNull String message) {
```

Comment:
I would also add the tag as a parameter, it's easier to find the source if you add a tag at each location, but it would make this function useless.

I don't see any necesity for this function though.